### DISCUSSION TITLE
Cannot find the production files

### DISCUSSION THREAD

**Discussion by adamsimonini** [wet-boew/wet-boew#9802] [Q&A]
I feel really dumb asking about this...

But when I follow the documentation to the downloads, I get to here:

![image](https://github.com/user-attachments/assets/498f62c9-6e1a-428d-a0ee-b15699ce62b1)

Clicking that gets me to the releases on Github. The releases, however, are the entire Source Folder files, including the Docker image, and all sorts of tooling for the repo. And sadly, there is no wet-boew.min.js within that code, which to my understanding is what a project need to become wet-boew enabled.

This is extremely confusing. My expectation is that I can directly download the requisite js and css files so that I can add wet-boew to my project. Instead, when I go downloads > releases, I get  the entire project without the built files I am looking for.

For example, when I go to "Downloads" on the jQuery website, I get a download button that leads me directly to the minified js content I need: https://jquery.com/download/

What am I not understanding?

**Comment by mercury64** [comment]
You need to download the distribution package. 
Sounds like you downloaded the source package.

At the very bottom of the download page:

<img width="1259" alt="Screenshot 2024-09-04 at 08 00 07" src="https://github.com/user-attachments/assets/a540bd6b-5060-4667-ab63-d643b22554d2">

<img width="809" alt="Screenshot 2024-09-04 at 07 59 25" src="https://github.com/user-attachments/assets/0aa3cc43-db6a-4658-8991-2d3b1f4131c8">
